File: Teledyne DALSA/Sapera/Demos/NET/SeqGrabDemo/CSharp/SeqGrabDemo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;

namespace DALSA.SaperaLT.Demos.NET.CSharp.SeqGrabDemo
{
    static class Program
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            SeqGrabDemoDlg SeqGD = new SeqGrabDemoDlg();
            if (!SeqGD.IsDisposed)
                Application.Run(SeqGD);
        }
    }
}

File: Teledyne DALSA/Sapera/Demos/NET/SeqGrabDemo/VB/SeqGrabDemo.vb
Public Class SeqGrabDemo
    'The main entry point for the application.
    <STAThread()> _
    Shared Sub Main()
        Application.EnableVisualStyles()
        Application.SetCompatibleTextRenderingDefault(False)
        Dim SeqGD As SeqGrabDemoDlg = New SeqGrabDemoDlg()
        If Not SeqGD.IsDisposed Then
            Application.Run(SeqGD)
        End If
    End Sub
End Class
